docs/source/advanced.rst

```diff
@@ -229,11 +329,61 @@ type, and implements basic handling of the following cases out of the box:
 |                          | [.2, .9]]    |                |                |               |
 +--------------------------+--------------+----------------+----------------+---------------+
 
-If you find that your target is classified as ``"multiclass-multioutput"`` or ``"unknown"``, you will have to
-implement your own data processing routine.
+The supported cases are handled by the default implementation of ``target_encoder``.
+The default implementations are available for use as :py:class:`scikeras.utils.transformers.ClassifierLabelEncoder`
+and :py:class:`scikeras.utils.transformers.RegressorTargetEncoder` for
+:py:class:`scikeras.wrappers.KerasClassifier` and :py:class:`scikeras.wrappers.KerasRegressor` respectively.
 
-For a complete examples implementing custom data processing, see the examples in the
-:ref:`tutorials` section.
+As per the table above, if you find that your target is classified as
+``"multiclass-multioutput"`` or ``"unknown"``, you will have to implement your own data processing routine.
+
+Whole dataset manipulation via data_transformer
```

Comment:
I think this section should go directly below the paragraph on dataset transformers.

Reply:
I'm also unsure about the level of nesting here. It looks like "multi-input" and "whole dataset" are on the same level, something I wouldn't have expected with the titles.

It might be worth collapsing this section into the "data transformers" section. This section really only mentions the signature.

Reply:
I'll move it as you suggest. It then becomes duplicated with the paragraph discussed in https://github.com/adriangb/scikeras/pull/167#discussion_r565614539, which warrants removing the latter.